server/src/main/java/io/pravega/schemaregistry/server/rest/auth/AuthHandlerManager.java

```diff
@@ -102,7 +123,7 @@ void registerHandler(AuthHandler authHandler) {
      * Context class which sets the context for a request. It is derived by using the auth handler for the authentication 
      * method and stores the authentication token.  
      */
-    public static class Context {
+    public static class Context implements SecurityContext {
```

Comment:
This class doesn't appear to belong here IMO. Can we move this outside of the `AuthHandlerManager`?